src/noc/guards/noc-guard.spec.ts

```diff
@@ -32,8 +32,9 @@ describe('NocGuard', () => {
         'pui-finance-manager',
         'pui-organisation-manager',
         'pui-user-manager'
-      ],
-    }
+      ]
+    },
+    locationInfo: []
```

Comment:
this should be optional and don't change NoC code

Reply:
Ok I get it what you mean if optional, We dnt need to change any. Agreed

Reply:
Done